test/functional/chrome-e2e-specs.js

```diff
@@ -0,0 +1,32 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import AndroidDriver from '../..';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+const capabilities = {
+  "browserName": "chrome",
+  "avd": "Nexus_5_API_25",
```

Comment:
no idea how are you guys managing e2e tests, but yeah a `process.env` var it's a good idea